assets/js/components/DashboardNavigation.js

```diff
@@ -200,24 +204,34 @@ export default function DashboardNavigation() {
 
 			const { hash } = global.location;
 			if ( closestID !== hash?.substring( 1 ) ) {
-				trackEvent(
-					`${ viewContext }_navigation`,
-					'tab_scroll',
-					closestID
-				);
+				if ( trackScrollEventRef.current ) {
+					trackEvent(
+						`${ viewContext }_navigation`,
+						'tab_scroll',
+						closestID
+					);
+				}
 
 				global.history.replaceState( {}, '', `#${ closestID }` );
 				setSelectedID( closestID );
 			}
 		};
 
 		const throttledOnScroll = throttle( onScroll, 50 );
+
+		const onScrollEnd = debounce( () => {
+			trackScrollEventRef.current = true;
+		}, 500 );
+
 		global.addEventListener( 'scroll', throttledOnScroll );
+		global.addEventListener( 'scroll', onScrollEnd );
```

Comment:
@jimmymadon  How about, instead of adding `onScrollEnd` as a separate event listener, just call it at the end of `onScroll` instead, thus reducing our event listener count?

Reply:
Thanks for taking a look at that @jimmymadon. I hadn't noticed this problem when trying out locally but can see how it would be an issue as you've described it. In that case, I am happy to approve this as-is.